contracts/Trader.sol

```diff
@@ -63,6 +66,9 @@ contract Trader {
 
         require(n > 0, "TDR: Received empty arrays");
 
+        // gas cost for individual traders
+        uint256 gasCost = (tx.gasprice) / (2 * n);
```

Comment:
think this should be `(tx.gasprice * tx.gasused) / (2 * n)`. I assume gas price is just the gas price in wei?

Reply:
Good catch!

Reply:
@raymogg Consult 6538408.